featurebyte/service/offline_store_feature_table.py

```diff
@@ -250,3 +252,27 @@ async def update_online_last_materialized_at(
             update_schema,
             document=document,
         )
+
+    async def list_precomputed_lookup_feature_tables(
+        self,
+        source_feature_table_id: ObjectId,
+    ) -> AsyncIterator[OfflineStoreFeatureTableModel]:
+        """
+        Retrieve list of precomputed lookup feature tables associated with a source feature table
+
+        Parameters
+        ----------
+        source_feature_table_id: ObjectId
+            Feature table id to search for
+
+        Yields
+        -------
+        AsyncIterator[OfflineStoreFeatureTableModel]
+            List query output
+        """
+        async for doc in self.list_documents_iterator(
+            query_filter={
+                "precomputed_lookup_feature_table_info.source_feature_table_id": source_feature_table_id
+            }
+        ):
+            yield doc
```

Comment:
Since we skip building precompute feature table for those feature lists do not require additional lookups, Just double whether we don't have to include source feature table here.

Reply:
It is intended for this method to return only the additional lookup tables, or return an empty list for source tables that do not require additional lookups.